lambdas/llama-2-13b-chat-llm/template.yml

```diff
@@ -0,0 +1,112 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: QnABot on AWS LLM Plugin for Llama-2-13b-chat
+
+Parameters:
+
+  LLMModel:
+    Type: String
+    Default: jumpstart-dft-meta-textgeneration-llama-2-13b-f
+    AllowedValues:
+      - jumpstart-dft-meta-textgeneration-llama-2-13b-f
+    Description: Llama 2 13b Chat LLM Model
+
+Resources:
+  LambdaFunctionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: lambda.amazonaws.com
+            Action: sts:AssumeRole
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+      Policies:
+        - PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "sagemaker:InvokeEndpoint"
+                Resource:
+                  - "*"
```

Comment:
Scope this to be least privilege..  Lambda only needs to invoke the specific endpoint specified as a parameter eg `SageMakerEndpointArn`

Reply:
Addressed in the 2nd commit.